catalog/policies/git_push.proposed-run.rego

```diff
@@ -27,9 +27,17 @@ ignore  {
 # Ignore draft PRs unless they explicitly have a `spacelift-trigger` label
 ignore {
    input.pull_request.draft
+   # we can only check if labels contains something if there is more than one label
+   count(input.pull_request.labels) > 0
```

Comment:
This line is probably not needed here since the line below expects there to be more than one value in `labels`